README.md

```diff
@@ -9,44 +10,60 @@
 
 [Discord]: https://planetarium.dev/discord
 
-> [!TIP]
-> If you're new to Nine Chronicles, try to visit our **Developer Portal**!
->
-> https://nine-chronicles.dev/
+> ✨ **Tip:**  
```

Comment:
How do you think about this change? @boscohyun @Atralupus 

Personally, I like GitHub-flavored markdown.